src/library/coverartdelegate.cpp

```diff
@@ -126,8 +125,7 @@ void CoverArtDelegate::paintItem(
         VERIFY_OR_DEBUG_ASSERT(m_pCache) {
             return;
         }
-        const double scaleFactor =
-                getDevicePixelRatioF(qobject_cast<QWidget*>(parent()));
+        const double scaleFactor = qobject_cast<QWidget*>(parent())->devicePixelRatio();
```

Comment:
Why `devicePixelRatio()` instead of `devicePixelRatioF()` as before?

Reply:
an oversight, good catch

Reply:
fixed